website/client/components/inventory/stable/index.vue

```diff
@@ -430,13 +430,16 @@
       mousePosition: MouseMoveDirective,
     },
     data () {
+      let savedSelectedSortBy = 'standard';
+      if (localStorage.getItem('selectedSortBy')) savedSelectedSortBy = JSON.parse(localStorage.getItem('selectedSortBy'));
```

Comment:
instead of calling getItem twice you could do something like this

```
const savedSelectedSortBy = localStorage.getItem('selectedSortBy') || 'standard';
```

Also I feel like the key where this is saved (selectedSortBy) should be a bit more descriptive, something like `habitica-stable-sort-preference`